### PR TITLE
fix color for outgoing failed

### PR DESCRIPTION
**Pull request by michaelkirk-signal**
Moving a bit closer to the spec

PTAL @charlesmchen

### CODE REVIEW

File: Signal/src/ViewControllers/ConversationView/Cells/OWSMessageFooterView.m [hunk 1]
```diff
@@ -164,12 +164,15 @@ - (void)configureLabelsWithConversationViewItem:(ConversationViewItem *)viewItem
...
     if ([self isFailedOutgoingMessage:viewItem]) {
-        self.timestampLabel.text
+        timestampLabelText
             = NSLocalizedString(@"MESSAGE_STATUS_SEND_FAILED", @"Label indicating that a message failed to send.");
     } else {
-        self.timestampLabel.text = [DateUtil formatMessageTimestamp:viewItem.interaction.timestamp];
+        timestampLabelText = [DateUtil formatMessageTimestamp:viewItem.interaction.timestamp];
     }
+
+    self.timestampLabel.text = timestampLabelText.localizedUppercaseString;
```
Comment (charlesmchen-signal) [inline Signal/src/ViewControllers/ConversationView/Cells/OWSMessageFooterView.m]: Hmm, there's a lot of places we're just using `uppercaseString`.  Can we please replace them with `localizedUppercaseString` - or make a card?

File: SignalMessaging/utils/ConversationStyle.swift [hunk 2]
```diff
@@ -137,7 +137,7 @@ public class ConversationStyle: NSObject {
     private static let defaultBubbleColorIncoming = UIColor.ows_messageBubbleLightGray
 
     @objc
-    public let bubbleColorOutgoingUnsent = UIColor.ows_light10
+    public let bubbleColorOutgoingFailed = UIColor.ows_darkSkyBlue
```
Comment (charlesmchen-signal) [inline SignalMessaging/utils/ConversationStyle.swift]: Huh, so it's now the same as the "sent" color?

Comment (michaelkirk-signal) [reply SignalMessaging/utils/ConversationStyle.swift]: According to zeplin.

File: Signal/src/ViewControllers/ConversationView/Cells/OWSMessageFooterView.m (hunk 1 above)
Comment (michaelkirk-signal) [reply Signal/src/ViewControllers/ConversationView/Cells/OWSMessageFooterView.m]: I've changed to localizedUppercaseString everywhere it makes sense.

**Pull request merged**